webapp/topics/topics.json

```diff
@@ -22,6 +22,18 @@
       "storage"
     ]
   },
+    {
+    "name": "Charmed Tempo HA",
+    "slug": "charmed-tempo-ha",
+    "topic_id": 15531,
+    "description": "A solution to deploy and operate Grafana Tempo, a distributed tracing backend that natively integrates with COS Lite, on Juju.",
+    "categories": [
+      "cloud",
+      "tracing",
```

Comment:
uhh, can we split the logging-tracing category in two instead? not sure what "logging-tracing" is.

Reply:
yeah I get that. I'm asking if it would make sense to remove the `logging-tracing` filter and tag, and instead add two separate ones: a `logging` and a `tracing`. Given that `tracing` is not applicable to Cos-Lite either way...

Reply:
done!